docs/user-guide/advanced/Pandas_API.ipynb

```diff
@@ -2507,6 +2507,46 @@
     "tab.prod(numeric_only=True)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Table.nunique()\n",
+    "```\n",
+    "Table.nunique(axis=0, skipna=True, numeric_only=False, min_count=0)\n",
+    "```\n",
+    "\n",
+    "Returns the number of unique elements across the given axis.\n",
+    "\n",
+    "**Parameters:**\n",
+    "\n",
+    "| Name         | Type | Description                                                                      | Default |\n",
+    "| :----------: | :--: | :------------------------------------------------------------------------------- | :-----: |\n",
+    "| axis         | int  | The axis to calculate the sum across 0 is columns, 1 is rows.                    | 0       |\n",
+    "| dropna       | bool | Don’t include NaN in the counts.                                                 | True    |\n",
```

Comment:
They like null in stead of NaN. But if this is a quote from another part of the documentation I would leave it as is